code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -86,7 +86,7 @@
 
 /obj/item/weapon/gun/projectile/automatic/ak922
 	name = "AK-922"
-	desc = "A New-Russia standard-issue battle rifle chambered in 7.62x39mm. Packs a punch and is built out of strong materials with an old yet reliable build."
+	desc = "A New-Russia standard-issue assault rifle chambered in 7.62x39mm. Packs a punch and is built out of strong materials with an old yet tried and proven build."

```

Comment:
Because a battle rifle is generally .308 caliber or 7.62x51mm caliber, which this gun is not. This is really just a correction on my -own- piece of work.